assets/ca-operator/ca-operator-deployment.yaml

```diff
@@ -67,17 +67,22 @@ spec:
                sleep 30
                continue
             fi
-            if ! oc get cm -n openshift-config-managed router-ca -o jsonpath='{ .data.ca-bundle\.crt }' > /tmp/router.ca; then
-               echo "Cannot fetch router-ca yet. Will continue to wait"
-               sleep 30
-               continue
-            fi
             if ! oc get cm -n openshift-config-managed service-ca -o jsonpath='{ .data.ca-bundle\.crt }' > /tmp/service.ca; then
                echo "Cannot fetch service-ca yet. Will continue to wait"
                sleep 30
                continue
             fi
-            cat /etc/kubernetes/config/initial-ca.crt /tmp/router.ca /tmp/service.ca > /tmp/kcm.ca
+            include_router_ca=1
+            if ! oc get cm -n openshift-config-managed router-ca -o jsonpath='{ .data.ca-bundle\.crt }' > /tmp/router.ca; then
```

Comment:
is the network error failure case here critical? This won't affect IKS but mainly for your other offering I could see that being a problem if the router cert is removed. 

Could wait and see if it's an issue but just something I noticed.

Reply:
> is the network error failure case here critical? This won't affect IKS but mainly for your other offering I could see that being a problem if the router cert is removed.

Because we run this continuously, we should be able to recover from failures in the next go-around.